src/Core/Migration/Migration1754897550AddFieldsToMigrationLogs.php

```diff
@@ -20,6 +21,8 @@
 #[Package('fundamentals@after-sales')]
 class Migration1754897550AddFieldsToMigrationLogs extends MigrationStep
 {
+    use ColumnExistsTrait;
```

Comment:
This shouldn't be necessary as it's already part of `MigrationStep` (which uses `AddColumnTrait`, which itself uses `ColumnExistsTrait`)